cmd/minikube/cmd/config/enable.go

```diff
@@ -52,6 +53,21 @@ var addonsEnableCmd = &cobra.Command{
 		if addon == "olm" {
 			out.Styled(style.Warning, "The OLM addon has stopped working, for more details visit: https://github.com/operator-framework/operator-lifecycle-manager/issues/2534")
 		}
+		addonBundle, ok := assets.Addons[addon]
+		if ok {
+			maintainer := addonBundle.Maintainer
+			if maintainer == "Google" || maintainer == "Kubernetes" {
+				out.Styled(style.Tip, `{{.addon}} is an addon maintained by {{.maintainer}}. For any concerns contact minikube on GitHub.
+    You can view the list of minikube maintainers at: https://github.com/kubernetes/minikube/blob/master/OWNERS`,
+					out.V{"addon": addon, "maintainer": maintainer})
+			} else if verifiedMaintainer := addonBundle.VerifiedMaintainer; verifiedMaintainer != "" {
+				out.Styled(style.Tip, `{{.addon}} is maintained by {{.maintainer}} for any concerns contact {{.verified_maintainer}} on GitHub.`,
+					out.V{"addon": addon, "maintainer": maintainer, "verified_maintainer": verifiedMaintainer})
```

Comment:
```suggestion
					out.V{"addon": addon, "maintainer": maintainer, "verifiedMaintainer": verifiedMaintainer})
					
```
Can we keep normal Go casing here

Reply:
of course